CHANGELOG.md

```diff
@@ -8,6 +8,11 @@
 #### Internal
 - Updated to XcodeProj 7.13.0 [#908](https://github.com/yonaskolb/XcodeGen/pull/908) @brentleyjones
 
+## 2.16.1
+
+#### Added
```

Comment:
This should probably be above `Fixed` in the `Next Version` section.

Reply:
@brentleyjones yeah, fixed)